tools/testbench/topology.c

```diff
@@ -767,6 +725,7 @@ static int load_filewrite(struct tplg_context *ctx, int dir)
 	register_comp(filewrite.comp.type, NULL);
 	if (ipc_comp_new(sof->ipc, ipc_to_comp_new(&filewrite)) < 0) {
 		fprintf(stderr, "error: new file write\n");
+		free(filewrite.fn);
```

Comment:
This took a while to understand, but seems to be a drive-by bugfix to an unrelated bug in the function.